### PR TITLE
Minor spelling and grammar edits

### PR DESCRIPTION
**Pull request by klyap**

### CODE REVIEW

File: documentations/mainpage.md [hunk 1]
```diff
@@ -126,22 +126,22 @@ User may use command line to unmount file system like this:
 If mounting is done with \ref DOKAN_OPTION_NETWORK, you need Dokan Network Provider for it to work correctly.
 This file (dokannp1.dll) *must* be copied to `%WINDIR%\system32` and you can register the provider on your system with `dokanctl.exe /i n` command.
 
-Without this [Network Provider](https://msdn.microsoft.com/en-us/library/windows/desktop/aa378776%28v=vs.85%29.aspx), Windows Explorer will not handle well Virtual drives mounted as network share and your drive could appears disconnected.
+Without this [Network Provider](https://msdn.microsoft.com/en-us/library/windows/desktop/aa378776%28v=vs.85%29.aspx), Virtual drives mounted as network share will not be handled well by Internet Explorer and your drive could appear disconnected.
 
-If Network Redirector is setup by setting up an UNC Name, Dokan Network Provider will assign UNC Name to the drive label automatically.
+If Network Redirector is setup by setting up an UNC Name, Dokan Network Provider will assign an UNC Name to the drive label automatically.
```
Comment (Rondom) [inline documentations/mainpage.md]: I think "a UNC" is correct. It's the pronunciation that determines a/an.

File: documentations/mainpage.md [hunk 2]
```diff
@@ -126,22 +126,22 @@ User may use command line to unmount file system like this:
...
+There is also [WinFSTest](https://github.com/Liryna/winfstest). Dokany is automatically running it on the sample mirror at every commit with appveyor.
 
 For C# developers, there is [DokanNet.Tests](https://github.com/dokan-dev/dokan-dotnet/tree/master/DokanNet.Tests) made by viciousviper that currently only test C# Mirror sample but can easily be changed to test a C# Dokan FS.
 
 ## Misc
 
 If there are bugs in Dokan library or file system applications which
 use the library, you will get the Windows blue screen. Therefore, it
-is strongly recommended to use Virtual Machine when you develop file
+is strongly recommended to use a Virtual Machine when you develop file
```
Comment (Rondom) [inline documentations/mainpage.md]: While you are at it: "virtual machine" (it's not a name, so small letters)

File: documentations/mainpage.md (hunk 1 above)
Comment (gkotian) [reply documentations/mainpage.md]: That is correct. Although `U` is a vowel, `a UNC` is correct in this context. A related article is [here](https://www.ecenglish.com/learnenglish/when-use-a)

**Comment by Liryna** [comment]
Hi @klyap ,

Thank you for your review and for taking the time ! 🥇 
This is a very appreciate work !

I have rebased your commits in a single one and merged it
https://github.com/dokan-dev/dokany/commit/2b447351c055623b5dfebe2fe075ab9bb5719d7d

**Pull request closed without merging**